packages/fuels-abigen-macro/tests/harness.rs

```diff
@@ -671,6 +672,38 @@ async fn type_safe_output_values() {
     assert!(response.value.bar);
 }
 
+#[tokio::test]
+async fn panics_on_non_binary_file() {
```

Comment:
Those tests are unit tests so you should probably put them in `contract.rs`

Reply:
Should I put the tests inside of contract.rs itself?  Or should I put the tests inside of a tests/ folder similar to how fuels-abigen-macro is structured?

Reply:
We generally put unit tests in the file containing the function they are unit-testing! So you can go ahead and put it directly in `contract.rs`.